### ISSUE TITLE
Combined Units

### ISSUE THREAD

**Issue by kosti1992** [nest/nestml#345]
Hello, 
there seems to be a problem with the processing of combined units, e.g. mV/ms. Currently, when parsing a model with such a unit, only the mV part is considered, while information about ms is dropped. A model which contains such a combined unit is, for example, aeif_cond_alpha. @PTraeder , it seems that you are working on this part of the framework, could you please explain if it is an error or some mistake on my side.

**Comment by PTraeder** [comment]
Those should work just fine. What makes you think they do not? Can you elaborate on what problem you are seeing?

**Comment by kosti1992** [comment]
@PTraeder Hey, I just reviewed my problem once more, and it seems that I read out the dimensions in a wrong way. The string array containing the powers of the base units consists of eight elements, where the last one is the overall magnitude. In fact, I did not see that it consists of eight elements instead of only seven (SI base units), and assumed that it has the same structure (and order) as given in the Builder class located in the UnitRepresentation. I am sorry for the inconvenience.

Greetings
Konstantin
